prow/plugins/config.go

```diff
@@ -1806,6 +1810,53 @@ func (p *Plugins) mergeFrom(other *Plugins) error {
 	return utilerrors.NewAggregate(errs)
 }
 
+func (p *Bugzilla) mergeFrom(other *Bugzilla) error {
+	if other == nil {
+		return nil
+	}
+
+	var errs []error
+	if other.Default != nil {
+		if (*p).Default != nil {
+			errs = append(errs, errors.New("configuration of global default defined in multiple places"))
+		} else {
+			(*p).Default = other.Default
+		}
+	}
+	if len(other.Orgs) != 0 && (*p).Orgs == nil {
+		newConfig := (*p)
+		newConfig.Orgs = make(map[string]BugzillaOrgOptions)
+		(*p) = newConfig
+	}
+	for org, orgConfig := range other.Orgs {
+		if _, ok := (*p).Orgs[org]; !ok {
+			(*p).Orgs[org] = BugzillaOrgOptions{}
+		}
```

Comment:
Golang automatically dererences points for you if you access their properties, can we just use that (here and everywhere else)? I had to read this a couple of times to be sure you didn't want to do something special 
```suggestion
		if _, ok := p.Orgs[org]; !ok {
			p.Orgs[org] = BugzillaOrgOptions{}
		}
```